src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -331,24 +331,493 @@ The following workflows represent some mathematical functions that can be optimi
 
 *Himmelblau_Function:* is a multi-modal function, used to test the performance of optimization algorithms.
 
-//.Himmelblau_Function_Workflow variables
-//[cols="2,5,2"]
-//|===
-//| *Variable name* | *Description* | *Type*
-//| `DOCKER_ENABLED`
-//|If True,  the workflow will be executed on a Docker container. link:https://hub.docker.com/r/activeeon/dlm3/[activeeon/dlm3] image is used by default. If False, the required libraries should be installed in the different nodes that will be used.
-//| Boolean (default=True)
-//| `DOCKER_IMAGE`
-//| Specifies the docker image that will be used to execute the task.
-//| String (default="activeeon/dlm3")
-//| `INPUT_VARIABLES`
-//| Specifies the x and y values.
-//| JSON format (default={"x": 3.5976705887960527, "y": -1.8471754188451843})
-//| `TOKEN`
-//|  Specifies the id for the tested parameters.
-//| String (default=-1)
-//|===
+== Model as a Service
 
+Using MLOS, you can deploy any machine learning model as a web service in any compute resource deployed on Proactive.
+
+The following deployment steps are similar wherever you need to deploy your model:
+
+. <<Start a Model Service>>
+. <<Deploy a Model>>
+. <<Call the service for predictions>>
+. <<Delete the service>>
+
+=== Start a Model Service
+
+There is two ways to start a web service (i.e. where you will deploy, update and call your model) using MLOS:
+
+*Via Cloud Automation portal*
+
+Open the link:https://try.activeeon.com/automation-dashboard/#/portal/cloud-automation[Cloud Automation Portal].
+
+Search for `Model_Service` in Services Workflows List.
+
+Set the following variables :
+
+.Model_Service variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `DOCKER_IMAGE`
+| the Docker image that contains all the dependencies and libraries needed for the model execution.
+| String (default="activeeon/dlm3")
+| `INSTANCE_NAME`
+| the name of the service that will be deployed.
+| String (default="model-server")
+| `YAML_FILE`
+| A YAML file that describes the OpenAPI Specification (known as Swagger Spec) of the service. This file should be stored in the catalog under the `model_as_service_resources` bucket. This file should be stored in the Catalog. More information about the structure of this file can be found in the section <<Customize the Service>>.
+| String (default="ml_service-api").
+| `PYTHON_ENTRYPOINT`
+| This entry script starts the service and defines the different functions to deploy the model, score the requests by using the model, and return the results. This specific script is specific to your model. This file should be stored in the Catalog. More information about this file can be found in the <<Customize the Service>> section.
```

Comment:
`This specific script is specific to your model.` => `This script is specific to your model.`